README.md

```diff
@@ -233,6 +233,15 @@ If you used Windows and decided to switch to Linux, this course should help you
 * [Bash Guide for Beginners](https://linux.die.net/Bash-Beginners-Guide/)
 * [Advanced Bash-Scripting Guide](https://linux.die.net/abs-guide/)
 
+### ZSH Useful Resources
+* [antigen](https://github.com/zsh-users/antigen)
+* [oh-my-zsh](http://ohmyz.sh/)
+* [Outrageously Useful Tips To Master Your Z Shell](http://reasoniamhere.com/2014/01/11/outrageously-useful-tips-to-master-your-z-shell/)
+* [awesome-zsh-plugins-list](https://github.com/unixorn/awesome-zsh-plugins)
```

Comment:
Can you add `awesome-zsh-plugins-list` either directly below the heading or at the end of this section? See what I did in the applications section for the reference: https://github.com/aleksandar-todorovic/awesome-linux#gui-based-applications